src/progressbar/progressbar.js

```diff
@@ -54,6 +54,7 @@ angular.module('ui.bootstrap.progressbar', [])
         require: '^progress',
         scope: {
             value: '=',
+            max: '=?',
```

Comment:
Yep, if it is used like that it would probably work. I can test that later today.
But isn't the "&" a one-way binding and @ a kind of "string-only" binding?